### ISSUE TITLE
Complex stable neural networks integration

### ISSUE THREAD

**Issue by VenuGopalVasarla** [VenuGopalVasarla/DCGAN-pytorch-venu#13]
Use https://github.com/soumith/ganhacks repo to make stable networks.

- [ ] Add complex discriminator and generator.

- [ ] Add new loss functions.

- [ ] Use new optimizers if possible
